2022.8/staged/core/data.yaml

```diff
@@ -1,5 +1,5 @@
 meta:
   phone_home: false
-run: []
-# python -c "import datetime; print(datetime.datetime.utcnow().strftime('%Y.%m.%d.%H.%M.%S'))"
```

Comment:
Well, I guess that's why it isn't the file already.
cc @lizgehret @Keegan-Evans @Oddant1